.github/workflows/verifyimage.yml

```diff
@@ -60,6 +65,18 @@ jobs:
           load: true
           push: false
 
+      - name: Scan ${{ matrix.target }}
+        uses: aquasecurity/trivy-action@0.27.0
+        with:
+          image-ref: ${{ needs.prepare.outputs.repo }}:${{ matrix.target }}
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
+          vuln-type: 'os,library'
+          severity: 'CRITICAL,HIGH,MEDIUM'
```

Comment:
I would say let's start with critical and highs.

Reply:
Sure, the scope has been narrowed down.